spec/labor/profile_image_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.describe ProfileImage, type: :labor do
+  let(:user) { create(:user) }
+
+  it "returns a profile_image" do
+    expect(described_class.new(user).get.size).to be > 0
+  end
```

Comment:
This is not testing that the get method actually returns what you expect, a better version would be:

```suggestion
  describe ".get" do
    it "returns a profile image" do
      expect(described_class.new(user).get).to eq(user.profile_image_url)
    end
  end
```